sln/OData.Pipeline.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30011.22
+# Visual Studio Version 17
+VisualStudioVersion = 17.8.34511.84

```

Comment:
Did we need to change VS version?

Reply:
I think this version refers to the last Visual Studio version that modified the file. It does not change the minimum Visual Studio version (see line 4). This file references Version 16 because we had never opened this solution in VS 2022 since this is mostly used in the build pipeline. But if you look at the `OData.sln` which we work with during development, the visual studio version is also 17.